src/devices/Adg731/samples/Adg731.Samples.BasicSample.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Configurations>Windows-Debug;Linux-Debug;Linux-Release;Windows-Release;Debug;Release</Configurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--Cases used when running in VS, with the solution config either Windows-Debug or Linux-Debug (or -Release) -->
+    <ProjectReference Condition="$(Configuration.Contains('Win'))" Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=win</AdditionalProperties>
+    </ProjectReference>
+    <ProjectReference Condition="$(Configuration.Contains('Linux'))" Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <!--Cases used when running build script in other cases. Just pick one -->
+    <ProjectReference Condition="'$(Configuration)' == 'Debug' or '$(Configuration)' == 'Release'" Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <ProjectReference Include="..\Adg731.csproj" />
+    <PackageReference Include="Iot.Device.Bindings" Version="1.0.0" />
```

Comment:
sample should follow naming conventions (including csproj name and content) as in other samples - this will currently have chicken and the egg problem: Iot.Device.Bindings doesn't have your library yet because we haven't shipped package yet and we won't until the build passes - this should do what every other project does: use project to project reference - ideally copy&paste content of other sample and modify as little as possible